test/Operators/hybrid/3d.jl

```diff
@@ -176,15 +176,31 @@ end
     hv_center_space, hv_face_space =
         hvspace_3D((-1.0, 1.0), (-1.0, 1.0), (-1.0, 1.0))
 
-    function rhs!(dudt, u, _, t)
+    abstract type BCtag end
+    struct ZeroFlux <: BCtag end
+    
+    bc_divF2C_bottom!(::ZeroFlux, dY, Y, p, t) =
+        Operators.SetValue(Geometry.UVWVector(0.0, 0.0, 0.0))
+    
+    struct ZeroFieldFlux <: BCtag end
+    function bc_divF2C_bottom!(::ZeroFieldFlux, dY, Y, p, t)
+        space = axes(Y.h).horizontal_space
+        FT = Spaces.undertype(space)
+        zeroflux = Fields.zeros(FT, space)
+        return Operators.SetValue(
+            Geometry.UVWVector.(zeroflux, zeroflux, zeroflux)
+        )
+    end
```

Comment:
@simonbyrne are there further unit tests we should add and is this an appropriate place to add this test?